packages/core/src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -221,10 +221,6 @@ export class RecommendationHandler {
                     'Invalid Request : ',
                     JSON.stringify(request, undefined, EditorContext.getTabSize())
                 )
-                const languageName = request.fileContext.programmingLanguage.languageName
-                if (!runtimeLanguageContext.isLanguageSupported(languageName)) {
-                    errorMessage = `${languageName} is currently not supported by Amazon Q inline suggestions`
-                }
```

Comment:
move this to upstream caller `recommendationService` so it can return early when the language is not supported